core/controllers/topic_editor.py

```diff
@@ -151,6 +152,23 @@ def get(self, topic_id: str) -> None:
 
         updated_canonical_story_summary_dicts = []
         for summary in canonical_story_summary_dicts:
+            story = story_fetchers.get_story_by_id(summary['id'])
+            nodes = story.story_contents.nodes
+            total_chapters_count = len(nodes)
+            published_chapters_count = len(
+                [node for node in nodes if
+                node.status == 'Published'])
```

Comment:
This and other instances should use a (reusable) named constant.

Reply:
Done